bot.py

```diff
@@ -25,5 +25,8 @@
     elif message.content.startswith('!sleep'):
         await asyncio.sleep(5)
         await client.send_message(message.channel, 'Done sleeping')
+    elif message.content.startswith('!d6'):
```

Comment:
@chasebaker288 This looks good!

There's a handy regular language we can recognize here, though.

Can you update this to match on `xdy`, and extract x and y from the regex results on a match?